src/argilla/utils/dependency.py

```diff
@@ -63,22 +63,29 @@ def _compare_versions(
         )
 
 
-def require_version(requirement: str, func_name: Optional[str] = None) -> None:
+def require_version(requirement: Union[str, List[str]], func_name: Optional[str] = None) -> None:
```

Comment:
Note that `func_name` should also be handled, so this may be tricky!

Plus ideally we can rename `func_name` to `fn_name` 👍🏻

Reply:
I will rename `func_name` to `fn_name`! Regarding the handling of the argument, I think we can keep discussing it on the previous conversation.